CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The most recent changes are on top, in each type of changes category.
 ### Added
 
 - Audit selected entities [#158](https://github.com/cyfronet-fid/sat4envi/issues/158)
+- Add /logout routing and clear storage on logout (inconsistent state of storage) [#445](https://github.com/cyfronet-fid/sat4envi/issues/445)
```

Comment:
changelog order. new entries on top.

Reply:
Is it influencing anything? Don't we deploy things in packages of features? Does it matter for the client, what was an order of merges?